documentation/API-GUIDELINES.md

```diff
@@ -41,8 +41,8 @@ The following paragraphs contain additional recommendations.
 
 ## API Surface
 
-- API documentation shouldn't be an afterthought.
-- Private details shouldn't leak into the public API, and should be made private where technically possible.
+- API documentation must be provided for every new driver and API.
+- Private details must not leak into the public API, and should be made private where technically possible.
```

Comment:
I think this can't me a "must not" because of the two exceptions listed :)

Reply:
Fair enough, I'll revert the wording here.